src/database/DataTypes/StocktakeItem.js

```diff
@@ -26,6 +34,16 @@ export class StocktakeItem extends Realm.Object {
     return this.item ? this.item.code : '';
   }
 
+  /**
+   * Check to see if any StocktakeBatches have been adjusted for this StocktakeItem
+   * Will return true if any StocktakeBatches were changed, even if net quantity change
+   * for StocktakeItem is 0.
+   * @return  {boolean} True if StocktakeBatches have adjustments
+   */
+  get hasBatchWithQuantityChange() {
+    return this.batches.some(stocktakeBatch => stocktakeBatch.difference !== 0);
```

Comment:
I've alluded to it elsewhere, but would it be better to check for `!stocktakeBatch.countedNumberOfPacks`. Check that for null means that where you use this getter, it'll correctly show a diff value of `0` where batches were counted but with no net change.

Reply:
This adds a bit more complexity.. in calculating difference and in calculating countedTotalQuantity.
Take this example: Two StocktakeBatches for one StocktakeItem
* No adjustment in StocktakeBatches, so StocktakeItem.difference === 0 and StocktakeItem.countedTotalQuantity === null (that's easy)
* Adjustment to one StocktakeBatch... StocktakeItem.countedTotalQuantity === (countedTotalQuantity of the changed StocktakeBatch and snapshotQuantity for the second StocktakeBatch, even though it's countedNumberOfPacks is null)

It's much easier to assume that countedNumberOfPacks === snapshotNumberOfPacks when a StocktakeBatch is created.